cocos/ui/UIButton.cpp

```diff
@@ -150,6 +150,7 @@ void Button::createTitleRenderer()
 {
     _titleRenderer = Label::create();
     _titleRenderer->setAnchorPoint(Vec2::ANCHOR_MIDDLE);
+    _titleRenderer->setAlignment(TextHAlignment::CENTER, TextVAlignment::CENTER);
```

Comment:
You can't change the default alignment to horizontal center and vertical center. It will break back compatibility.